File: selenium/src/java/org/openqa/selenium/internal/AltLookupStrategy.java
package org.openqa.selenium.internal;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

public class AltLookupStrategy implements LookupStrategy {
    public WebElement find(WebDriver driver, String use) {
        return driver.findElement(By.xpath("//*[@alt='" + use + "']"));
    }
}

File: dotnet/src/Selenium.WebDriverBackedSelenium/Internal/SeleniumEmulation/LookupStrategies/AltLookupStrategy.cs
using System;
using OpenQA.Selenium;

namespace Selenium
{
    internal class AltLookupStrategy : ILookupStrategy
    {
        public IWebElement Find(IWebDriver driver, string use)
        {
            return driver.FindElement(By.XPath("//*[@alt='" + use + "']"));
        }
    }
}
